app.js

```diff
@@ -13,324 +13,287 @@ if (!supportsLocalStorage()) {
+  let renderedInviteList = [];
+  //The local storage copy of storedInviteList should be updated to reflect the fullInviteList every time a change is made to it.
+  let storedInviteList = [];
+  //display gets marked up version of renderedInviteList each time a change is made to it. See reloadDOM.
+  let display = '';
 
-  //Stringify and add new names to the stored names list
-  function saveInviteeLIS(string) {
-    let inviteeLIS = getInviteeLIS();
-    if (!string || inviteeLIS.indexOf(string) > -1) {
-      return false;
-    }
-    inviteeLIS.push(string);
-    localStorage.setItem('inviteeLIS', JSON.stringify(inviteeLIS));
-    return true;
-  }
+  //Rewrites the local storage to sync up with the passed in list. Preferably always pass in full list not rendered.
+  const refreshLocalStorageList = (List) => {
+    localStorage.removeItem('storedInviteList');
+    localStorage.setItem('storedInviteList', JSON.stringify(List));
+  };
 
-  //Find and delete a name from the stored names list, then resave the list with the amended contents
-  function removeInviteeLIS(inviteeName) {
-    const currentStoredNames = getInviteeLIS();
-    for (i = 0 ; i < currentStoredNames.length ; i++) {
-      if (inviteeName === currentStoredNames[i]) {
-        currentStoredNames.splice(i , 1);
-        localStorage.removeItem('inviteeLIS');
-        for (i = 0 ; i < currentStoredNames.length ; i++) {
-          saveInviteeLIS(currentStoredNames[i]);
-        }
-      }
+  //Fetch the stored list and populate storedInviteList with it
+  const getLocalStorageList = () => {
+    storedInviteList = localStorage.getItem('storedInviteList');
+    if (storedInviteList) {
+      storedInviteList = JSON.parse(storedInviteList);
+      return storedInviteList;
     }
-  }
+  };
 
-  //Create and append div, label and checkbox for new filters
-  function createFilterDiv(labelName, checkboxName, divName, labelText) {
-    labelName.textContent = labelText;
-    checkboxName.type = 'checkbox';
-    divName.appendChild(checkboxName);
-    divName.appendChild(labelName);
-    filterConDiv.appendChild(divName);
-  }
+  //formats and prints each object in the rendered list to the screen
+  const reloadDOM = () => {
+  	renderedInviteList.map((current, index, arr) => {
+  		return display += `
+      <li id="${current.id}">
+        	<span class="inviteeName">
+            ${current.name}
+          </span>
+          <label>
+            Attending
+            <input id="setAttending" type="checkbox"
+            ${current.attending === true ? 'checked="true"' : null}>
+          </label>
+          <label>
+            Not Attending
+            <input id="setNotAttending" type="checkbox"
+            ${current.notAttending === true ? 'checked="true"' : null}>
+          </label>
+          <label>
+            Notes:
+            <${current.editState === 'Saved' ? 'p id="notes">' : 'input type="text" id="notes" value="' + current.notes + '">'}
```

Comment:
So es6 has this amazing thing called template strings. Try this:
``input type="text" id="notes" value="${current.notes}">``
You use a back tick and then ${} to insert js inside of a string.